internal/dataaccess/expense_dao.go

```diff
@@ -0,0 +1,161 @@
+package dataaccess
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Yokithaii/CLI_Expense_Tracker/internal/entities"
+	"github.com/jackc/pgx/v5"
+)
+
+type ExpenseDAO struct {
+	conn *pgx.Conn
+}
+
+func (e *ExpenseDAO) CloseConnect(context context.Context) {
```

Comment:
```suggestion
func (e *ExpenseDAO) CloseConnect(ctx context.Context) {
```

Для контекста есть устойчивая конвенция именования - `ctx`, стоит ей следовать.